docs/docs/components/divider.mdx

```diff
@@ -0,0 +1,99 @@
+# Divider
+
+### Quick start
+
+Here's a quick start guide to get started with the divider component
+
+### Importing Component
+
+```jsx
+import { Divider } from "@hover-design/react";
+```
+
+### Code Snippets and Examples
+
+##### Divider Default
+
+import "@hover-design/react/dist/style.css";
+import { Divider } from "@hover-design/react";
+
+```jsx
+<div className="App">
+  <Divider />
+</div>
+```
+
+> **Note:** Horizontal Divider's width will depend on container's width
+
+<Divider />
+
+##### Divider Dashed
+
+```jsx
+<div className="App">
+  <Divider type="dashed" />
+</div>
+```
+
+<Divider type="dashed" />
+
+##### Divider Dotted
+
+```jsx
+<div className="App">
+  <Divider type="dotted" />
+</div>
+```
+
+<Divider type="dotted" />
+
+##### Divider with Label
+
+```jsx
+<div className="App">
+  <Divider label="label" />
+</div>
+```
+
+<div style={{ marginTop: "30px" }}>
+  <Divider label="label" />
+</div>
+
+##### Vertical Divider
+
+```jsx
+<div className="App">
+  <Divider orientation="vertical" />
+</div>
+```
+
+> **Note:** Vertical Divider's height will depend on container's height
+
+<div style={{ height: "100px", margin: "0 auto", width: "fit-content" }}>
+  <Divider orientation="vertical" />
+</div>
+
+##### Vertical Divider with Label
```

Comment:
can you please do an example with horizontal list and vertical divider?

should look like

```
Example 1 | Example 2 | Example 3
```

Reply:
sure